firefox_puppeteer/ui/about_window/window.py

```diff
@@ -0,0 +1,140 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+
+from marionette_driver import By, Wait
+
+from ..windows import BaseWindow
+from ...api.software_update import SoftwareUpdate
+from deck import Deck
+
+
+class AboutWindow(BaseWindow):
+    """Representation of the About window."""
+
+    TIMEOUT_UPDATE_APPLY = 300
+    TIMEOUT_UPDATE_CHECK = 30
+    TIMEOUT_UPDATE_DOWNLOAD = 360
+
+    window_type = 'Browser:About'
+
+    def __init__(self, marionette_getter, window_handle):
+        BaseWindow.__init__(self, marionette_getter, window_handle)
+
+        self.software_update = SoftwareUpdate(marionette_getter)
+        self.download_duration = -1
+
+    @property
+    def deck(self):
+        """The :class:`Deck` instance which represents the deck.
+
+        :returns: Reference to the deck.
+        """
+        deck = self.window_element.find_element(By.ID, 'updateDeck')
+        return Deck(lambda: self.marionette, self, deck)
+
+    @property
+    def patch_info(self):
+        """ Returns information about the active update in the queue.
+
+        :returns: A dictionary with information about the active patch
+        """
+        patch = self.software_update.patch_info
+        patch['download_duration'] = self.download_duration
+        return patch
+
+    @property
+    def updates_found(self):
+        """ Checks if updates were found.
+
+        :returns: True if updates were found
+        """
+        return self.wizard_state != 'noUpdatesFound'
```

Comment:
Imagine you open the about window and call this method. It will return true because the initial panel will be `!= 'noUpdatesFound'. The software update library is the place to query for an available update.